### PR TITLE
tinygo: fix typo

### PR DESCRIPTION
**Pull request by TrevorSmale**
## Description of changes

[- See diff.](tinygo: fix typo)

### CODE REVIEW

**Review by pbsds — APPROVED** [review]
LGTM, should cause [no conflict](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+%2Ftinygo-bootstap-tools%2F&type=code), just a tiny thing to follow nixpkgs convention

File: pkgs/development/compilers/tinygo/default.nix
Comment (pbsds) [inline pkgs/development/compilers/tinygo/default.nix:1]: Please reword the commit to `tinygo: fix typo`

**Review by pbsds — DISMISSED** [review]

**Review by wolfgangwalther — APPROVED** [review]
I suppose a committer could just squash and fix the commit message accordingly.

**Comment by pbsds** [comment]
Thanks for the bump, i have since become a committer.

**Comment by kirillrdy** [comment]
> Thanks for the bump, i have since become a committer.
> 
> waiting for eval

i am the author of the spelling mistake :-)

**Comment by kirillrdy** [comment]
I've updated PR title to match commit message

**Review by pbsds — APPROVED** [review]
Now i am confused. Tinygo fails on all platforms on hydra likely since https://github.com/NixOS/nixpkgs/pull/262346:

* https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.tinygo.x86_64-linux
* https://hydra.nixos.org/job/nixos/trunk-combined/nixpkgs.tinygo.aarch64-linux
* https://hydra.nixos.org/job/nixpkgs/trunk/tinygo.x86_64-darwin
* https://hydra.nixos.org/job/nixpkgs/trunk/tinygo.aarch64-darwin

They all fail fetching the source, across two version bumps.

ofborg aarch64-linux passes on this PR, but not x86_64-linux, due to a build failure. The installPhase warnings on aarch64-linux indicates that it only passes due to a what i assume is a `set +e` somewhere.

This PR should be good to merge however, since `tinygo-bootstrap-tools` at least seems to build, but tinygo really could use a look over.

**Comment by kirillrdy** [comment]
Result of `nixpkgs-review pr 257183` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tinygo</li>
  </ul>
</details>

**Pull request merged**